### ISSUE TITLE
Add option to calculate altitude for certain conditions

### ISSUE THREAD

**Issue by xZise** [ryohpops/kspRemoteTechPlanner#4]
Hi I'd like if there is an option that shows me the altitude at that certain conditions are met. For example I have 3 satellites around Kerbin defined with a Communotron 16 and I want the highest altitude at which those still have connectivity (so that I can cover the largest volume). Alternatively the lowest altitude at which the antennae (if my delta-v budget is restricted, although that is not really important). Another condition I'd like is the altitude at which the ground is still covered. So the altitude at which both spheres of neighboring satellites are intersecting at the ground.

**Comment by ryohpops** [comment]
Thank you for your suggestion, but I'm afraid I cannot accept them.

First, the highest and lowest altitude are of no practical use.
Because you cannot launch a satellite into these very altitudes in the in-game universe which exists on the physics engine.
The inaccuracy leads an accidental disconnection of your satellite communication.
I recommend you to change the planner's altitude by some convenient interval (like 10km, 100km, for readability) and find a good one which makes nice distance between two satellites compared to the equipped antenna's range.

Second, the altitude for ground communication is not necessary.
Basically, there are no connection-less area on the equator (or the orbit's inclination) as far as you are considering practical number of satellites. 
Only geographical features make the shade of communication, and I cannot take account of each mountain to calculate the altitude.
I think the lowest height of the communication line should be calculated. I'll add it to my todo list.

**Comment by xZise** [comment]
I know that the exact altitudes are not useful but it would give you a starting point. For example for the maximum altitude of 3 satellites it would return 8XX km so I could start with 800 km altitude and if I think there is enough buffer would use that.

And regarding ground communications someone (who actually made me aware of the tool) in the IRC posted that they launch 8 satellites. And with so many satellites you can get the craft far enough apart that you have coverage holes around the equator (the maximum altitude with a distance between them is > 2.6 Mm but you need 2.4 Mm-ish to cover the ground).

You could in theory make a minimum stable orbit circle like you already do a maximum stable orbit. By the way I don't think RT actually checks for terrain but just takes the body's radius in account.

**Comment by ryohpops** [comment]
OK, I understood their importance.
I will add them to my TODO list.

And your understanding is right, it is mentioned on the wiki that RT handles a body as a perfect sphere for calculation.
Thank you for the information.
